tests/test_transforms/test_missing_values/test_resample_transform.py

```diff
@@ -132,3 +136,16 @@ def test_get_regressors_info_not_fitted():
 def test_params_to_tune():
     transform = ResampleWithDistributionTransform(in_column="regressor_exog", distribution_column="target")
     assert len(transform.params_to_tune()) == 0
+
+
+def test_working_with_categorical_columns(example_tsds):
```

Comment:
Not sure that we need pipeline here, we just want to check that `fit_transform` works with categoricals. Try to be more minimalistic